tox.ini

```
@@ -24,7 +24,7 @@ deps = flake8
 skip_install = true
 
 [testenv:isort]
-commands = isort --recursive --check-only --diff
+commands = isort --check-only --diff
 deps = isort
```

As the CLI is backwards incompatible, we can add a version constraint.

```suggestion
deps = isort >= 5.0.0
```